src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/RecursiveSpinLockTest.kt

```diff
@@ -14,13 +14,15 @@ import org.jetbrains.kotlinx.lincheck.annotations.Operation
 import org.jetbrains.kotlinx.lincheck.checkImpl
 import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.ModelCheckingOptions
 import org.jetbrains.kotlinx.lincheck_test.util.checkLincheckOutput
+import org.jetbrains.kotlinx.lincheck_test.util.IgnoreInTraceDebuggerMode
 import org.junit.Test
 import java.util.concurrent.atomic.AtomicBoolean
 import java.util.concurrent.atomic.AtomicInteger
 
 /**
  * Checks proper output in case of recursive spin-lock in one thread.
  */
+@IgnoreInTraceDebuggerMode
```

Comment:
Can we not ignore but have a different output file when `inTraceDebuggerMode == true`? The same for all other tests.

Reply:
Most of the tests, failing because of LoopDetector disabling, fail with StackOverflow and other similar non-recoverable errors.